adam/curriculum/phase1_curriculum.py

```diff
@@ -2033,7 +2050,7 @@ def make_walk_run_template(
                             agent,
                             SpatialPath(
                                 operator=TOWARD,
-                                reference_object=goal,
+                                reference_source_object=goal,
```

Comment:
Similar comment to falling; I would think if the goal is where the walker ends up, it shouldn't be the source of the spatial path? Marking subsequent related comments with :book: